static/js/lib/ckeditor/plugins/ResourceEmbed.ts

```diff
@@ -14,7 +14,7 @@ import {
   RESOURCE_EMBED_COMMAND
 } from "./constants"
 
-export const RESOURCE_SHORTCODE_REGEX = /{{< resource (\S+) >}}/g
+export const RESOURCE_SHORTCODE_REGEX = /{{< resource "?([^\s"]+)"? >}}/g
```

Comment:
I would prefer to do this more positively, so I like `/"?([a-zA-Z0-9\-]+)"?/` better. But...

> Possibly we shouldn't worry about this since it's not clear what we'd want to have happen if there is an invalid UUID saved in the Markdown - invalidate the link? alert the user? try to fix it somehow? I'm not sure.

I think one thing that should happen is: The user should still be able to edit the rest of the content in this page. I.e.,

1. Markdown has `{{< resource df6cfc84-3d60-我的正则表达式-da5b-f981-5ea0a29bd9e6 >}}`
2. User opens page and makes unrelated edit, clicks save
3. The markdown should **still have** `{{< resource df6cfc84-3d60-我的正则表达式-da5b-f981-5ea0a29bd9e6 >}}` (Not some gnarly escaped stuff.) 

So, I propose keeping it this way.

Reply:
that makes sense I think, and we definitely want to avoid creating more character escaping issues!